pydoctor/templates/apidocs.css

```diff
@@ -195,30 +202,90 @@ tr.private {
     margin-bottom: 6px;
 }
 
+/* This is magic to show the transparent index link */
+a.navbar-brand {
+    /* background-color:#f3d3db; */
+    border:solid;
+    margin-right: 0.3em;
+    border-width: 1.5px;
+    border-color: #c7254e;
+    background-color:#f9f2f4;
+    border-radius: 4px;
+}
+
+/* This is for the main top navagation bar */
+.page-header > h1 > code {
+    background-color:#f9f2f4;
+    /* background-color:transparent; */
+}
+
+/* .page-header > h1 > code > a {
+    background-color:#FCF9FA;
+} */
+
+/* 
+<a><code></code><a> for links
+This is inline docstring content marked up as code, for example C{foo} in epytext
+*/
+a > code {
+    color:#c7254e; 
+    background-color:#f9f2f4;
+    /* background-color:transparent; */
+}
+/* 
+<a class="code"> links that are present in summary tables 
+*/
+a.code {
+    color:#c7254e; 
+    background-color:#f9f2f4;
+    /* background-color:transparent; */
+}
+/* 
+This defines  the "code" class, it's black on light gray
+*/
 .code {
     font-family: Menlo,Monaco,Consolas,"Courier New",monospace;
     padding:2px 4px;
     font-size:90%;
-    color:#c7254e;
-    background-color:#f9f2f4;
+    color: #222222;
+    background-color: #f9f2f4;
+    /* color:#c7254e; 
+    background-color:#FCF9FA;*/
     border-radius:4px
 }
+/* 
+This overwrites the default values inherited from bootstrap min
+*/
+code {
+    color: #222222;
+    background-color: #f9f2f4;
```

Comment:
I think it would look better if we used pink background for red text and gray background for near-black text.